src/hotspot/share/gc/shenandoah/shenandoahCardTable.cpp

```diff
@@ -23,6 +23,7 @@
  */
 
 #include "precompiled.hpp"
+#include "gc/shenandoah/shenandoahHeap.inline.hpp"
```

Comment:
What is this include used for?

Reply:
This include will be removed in the next commit.